src/temp/AssignedUsers/AssignUsers.css

```diff
@@ -0,0 +1,9 @@
+.AssignUsersWrapper {
```

Comment:
are all changes in tmp folder 100% the same as in ui-users?

Reply:
I guess 95% the same. I have updated the constants path and improved code styling according to eslint setup in the module.

Reply:
@NikitaSedyx ui-users story one-liner fixes included in this PR https://github.com/folio-org/ui-users/pull/2534/files#diff-b4f7d17e02e42601400baccbbc6118e2ea1ce097d40ffc093c028c9f68265ed2R38

Reply:
hm, what have you updated? these files should be 100% the same

Reply:
@NikitaSedyx you can refer to this PR https://github.com/folio-org/ui-users/pull/2535

Reply:
I can't, you mentioned that it's 95%, what is 5% difference? I can't compare files line by line

Reply:
or do you say that you replicated changes to ui-users as well?

Reply:
5% Change was only code styling and all the changes from the PR https://github.com/folio-org/ui-users/pull/2535. Now, it is 100% MATCH.